### ISSUE TITLE
Plug-in to show touch positions

### ISSUE THREAD

**Issue by zougloub** [WayfireWM/wayfire#2506]
It would be nice to have a plugin that can show touch positions on screen, to help with accessibility, or debug touchscreens.

It seems there's a crosshair plug-in in wayfire-plugins-extra that can show mouse pointer position, the idea would be to have something analogue for touchscreen devices.

**Comment by zougloub** [comment]
@soreau is implementing a showtouch plugin in https://github.com/WayfireWM/wayfire-plugins-extra.

We may move this issue there.

I tested the current status, where the showtouch plugin should also work in multi-display situations, and there's one bug : if I press on 2 screens at the same time, the purple dot is appearing only once, which I consider OK because I see this plugin as a debugging tool, but it's position doesn't respect the screen position, eg. in my case, if I put one finger on the right edge of the left screen, and one finger on the left edge of the right screen, the purple dot is at the right edge of the right screen, when it should be between the fingers.

**Comment by soreau** [comment]
> @soreau is implementing a showtouch plugin in https://github.com/WayfireWM/wayfire-plugins-extra.
> 
> We may move this issue there.
> 
> I tested the current status, where the showtouch plugin should also work in multi-display situations, and there's one bug : if I press on 2 screens at the same time, the purple dot is appearing only once, which I consider OK because I see this plugin as a debugging tool, but it's position doesn't respect the screen position, eg. in my case, if I put one finger on the right edge of the left screen, and one finger on the left edge of the right screen, the purple dot is at the right edge of the right screen, when it should be between the fingers.

I'm guessing this problem probably stems from the fact that the output layout is reversed in the config and there is some bug. For reference, the calculation for the center point of all fingers is handled by [wf-touch here](https://github.com/WayfireWM/wf-touch/blob/master/src/touch.cpp#L10).
